src/main/kotlin/com/seokjoo/todo/domain/entity/todouser/Money.kt

```diff
@@ -6,7 +6,7 @@ import jakarta.persistence.Embeddable
 
 @Embeddable
 class Money(
```

Comment:
```suggestion
@JvmInline
value class Money(val balance: Long) { }
```

갑자기 든 생각인데, `@Embedded` `@Embeddable` 대신 `value class` 사용하는 건 어떠세요?
Money는 타입 명시를 위한 클래스니까 `value class`가 되게 잘 어울릴 것 같아서요